lib/tasks/development.rb

```diff
@@ -50,14 +50,11 @@ def container_rt_args
   end
 
   def docker_rt_args
-    [
-      '--user', "#{user.uid}:#{user.gid}"
-    ].freeze
+    [ '--tmpfs', '/run', '-v', '/sys/fs/cgroup:/sys/fs/cgroup:ro'].freeze
```

Comment:
These won't work with Docker on Mac I don't think.  I will test it out but /run and /sys/fs don't exist on Macs.

Reply:
OK. It's what I had to work in WSL (Ubuntu 20). We could do a spot check for darwin in `/etc/os-release`?

Reply:
*face palm* I know why this works, Docker on Macs actually runs inside a Linux VM that just has access to like /Users and other key locations on the Mac to seem like you are running Docker natively.  I verified I could rebuild, start and exec into container.

Reply:
Does this work for you on Mac?

Reply:
In the sense I can start the container with those flags, yes.  But OnDemand itself is not functional.  I get "Internal Server Error" page with URL "http://localhost:8080/pun/sys/dashboard".

First thing I notice that is weird is my working directory is same as path outside but only has "ondemand" folder, but likely not the cause.

This is in Apache error logs:

```
[Fri May 20 19:35:18.772461 2022] [auth_openidc:error] [pid 74:tid 139943934396160] [client 172.17.0.1:59604] oidc_provider_static_config: could not retrieve metadata from url: http://localhost:5556/.well-known/openid-configuration
```

I think might have just been a race condition or me not waiting long enough.  I verified that URL works inside container and I now get Dex login page if I refresh my browser outside container.

Reply:
Yea that happens sometimes if you're too quick to request the page and dex isn't completely booted up yet.